### DISCUSSION TITLE
Opensuse TW rpm Package

### DISCUSSION THREAD

**Discussion by marknefedov** [luisbocanegra/kde-material-you-colors#134] [Show and tell]
Hello, I've tried to package your work to SUSE Tumbleweed. 
https://build.opensuse.org/package/show/home:MarkNefedov/kde-material-you-colors

Specfile can probably be altered to support Fedora too, but I've currently have no fedora system.

**Comment by luisbocanegra** [comment]
Fedora has packages named with prefix python3- and is missing python3-pywal and material-color-utilities
Tried a fedora vm and it built/installed with these dependencies:
```
Source:         kde-material-you-colors-1.5.1.tar.gz
Requires:       python3
Requires:       python3-dbus
Requires:       python3-numpy
Requires:       python3-pillow
Requires:       python3-regex
BuildArch:      noarch
```
material-color-utilities-python would have to be packaged too ~(or shipped inside package?)~, python-pywal is an optional dependency

**Reply by marknefedov** [reply]
I've packaged material-color-utilities-python and made pywal optional, should be possible install fine now. Thank you for checking out!

**Reply by luisbocanegra** [reply]
And you! :)

```sh
[luis@fedoraa37 ~]$ sudo dnf install kde-material-you-colors
Last metadata expiration check: 0:06:00 ago on Sun 18 Dec 2022 08:24:38 AM CST.
Error: 
 Problem: conflicting requests
  - nothing provides python3-Pillow needed by kde-material-you-colors-1.5.1-23.1.noarch
  - nothing provides python3-dbus-python needed by kde-material-you-colors-1.5.1-23.1.noarch
  - nothing provides python3-material-color-utilities-python needed by kde-material-you-colors-1.5.1-23.1.noarch
(try to add '--skip-broken' to skip uninstallable packages)
[luis@fedoraa37 ~]$
```
These should be `python3-pillow` `python3-dbus` `python-material-color-utilities-python` ;)

**Reply by marknefedov** [reply]
Oh sorry! I've overlooked it. 
I've just realized that I can use docker to and test installation, at least in container 😄.
Anyway, looks like it installs.
![image](https://user-images.githubusercontent.com/20001291/208313689-cbfdbabf-a928-4cf9-98fe-7ea272d7403e.png)

**Reply by luisbocanegra** [reply]
Works! 🙌
![image](https://user-images.githubusercontent.com/15076387/208316563-2d15e342-fdc8-441f-839a-2b40cb709f0b.png)

**Comment by luisbocanegra** [comment]
Hi,
Are you still interested in maintaining this package?

I noticed you haven't updated it yet to the latest version and a user reported dependency problems for OpenSUSE here https://github.com/luisbocanegra/kde-material-you-colors/issues/160.

I have converted the project into a python package so it _should_ be easier to package now

If for whatever reason you can't maintain this package anymore is completely fine, just let me know, so I can update the readme accordingly.

**Reply by marknefedov** [reply]
Sorry, life happened. I do indeed have no system and time to test and maintain package. If somebody wants to do it they can fork it on OBS.

**Reply by marknefedov** [reply]
I'll try to update it at least, but no guarantees.

**Reply by luisbocanegra** [reply]
No worries if you don't, I appreciate that you took the time and effort to do this, and I am sure someone will continue it they find it useful.
